rc/core/kakrc.kak

```diff
@@ -15,25 +15,38 @@ add-highlighter -group / regions -default code kakrc \
     comment (^|\h)\K\# $ '' \
     double_string %{(^|\h)"} %{(?<!\\)(\\\\)*"} '' \
     single_string %{(^|\h)'} %{(?<!\\)(\\\\)*'} '' \
-    shell '%sh\{' '\}' '\{'
```

Comment:
Considering how many combinations are possible, maybe it would be worth trying to generate those lines in a shell scope? Or do we consider that those are the most commonly used delimiters and we don't want to be exhaustive anyway?